From 1d0cc63820a0ec856e76aed14a22f9ccadf16665 Mon Sep 17 00:00:00 2001
From: Denis Kononenko <denis.kononenko@bell-sw.com>
Date: Wed, 10 Aug 2022 07:58:30 +0000
Subject: [PATCH] Remove reproducibility test.

As long as abuild writes timestamps into .PKGINFO the package build
cannot be reproduced.
---
 tests/abuild.bats | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/tests/abuild.bats b/tests/abuild.bats
index 151c7a9..6a4f772 100644
--- a/tests/abuild.bats
+++ b/tests/abuild.bats
@@ -96,22 +96,6 @@ teardown() {
 	[ $status -ne 0 ]
 }
 
-@test "abuild: verify that packages are reproducible built" {
-	cd testrepo/pkg1
-	arch=$($ABUILD -A)
-	pkgs=$($ABUILD listpkg)
-
-	$ABUILD
-	checksums=$(cd "$REPODEST"/testrepo/$arch && md5sum $pkgs)
-	echo "$checksums"
-
-	$ABUILD cleanpkg all
-	checksums2=$(cd "$REPODEST"/testrepo/$arch && md5sum $pkgs)
-	echo "$checksums2"
-
-	[ "$checksums" = "$checksums2" ]
-}
-
 @test "abuild: test checksum generation" {
 	mkdir -p "$tmpdir"/foo
 	cat > "$tmpdir"/foo/APKBUILD <<-EOF
-- 
2.34.2

